MetricsUploader/proto/orbit_log_event.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package orbit_metrics_uploader;
+
+// This proto is used for communication between Orbit and metrics uploader
+// client. Changing this proto requires changing the metrics uploader client
+// library accordingly.
+message OrbitLogEvent {
+  enum LogEventType {
+    UNKNOWN_EVENT_TYPE = 0;
+    ORBIT_INITIALIZED = 1;
+    ORBIT_CAPTURE_DURATION = 2;
+  }
+  LogEventType log_event_type = 1;
+
+  string orbit_version = 2;
+  int64 event_duration_milliseconds = 3;
```

Comment:
Do all events necessarily have a duration? And maybe we want a timestamp of when an event occurred? That would allow computations of durations after the fact. E.g. types ORBIT_CAPTURE_STARTED and ORBIT_CAPTURE_FINISHED with a timestamp each. Or does that cause other problems in the analysis of the data?

Reply:
Not all the events have duration, this field is optional. There were 2 options that I've considered: to send event with duration or to send 2 events (start and end) and use sql query to calculate the duration. From the metrics uploader client perspective they look pretty similar, but I guess that having duration field would make queries easier (for example, when we want to get mean/max capture length for the specific time period). If we want to use ORBIT_CAPTURE_STARTED and ORBIT_CAPTURE_FINISHED only to calculate the duration I think it would be better just to send duration event, or am I missed something?